docker/scripts/import_db.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -o errexit
+cd `dirname $0`/../..
```

Comment:
Since this is only intended to run inside a container (it has hardcoded paths), let's just use `cd /home/musicbrainz/musicbrainz-server` here for clarity.